pkg/build/sbom.go

```diff
@@ -171,7 +172,18 @@ func readReleaseData(fsys fs.FS) (*osr.Data, error) {
 		return nil, fmt.Errorf("reading os-release: %w", err)
 	}
 
-	return osr.Parse(string(osReleaseData)), nil
+	releaseStr := string(osReleaseData)
+
+	// osr.Parse can panic if given improper input, so error early instead.
+	for _, line := range strings.Split(releaseStr, "\n") {
+		if line != "" {
+			if len(strings.Split(line, "=")) < 2 {
+				return nil, fmt.Errorf("invalid os-release line: %q", line)
```

Comment:
The format of this file is described at https://www.freedesktop.org/software/systemd/man/latest/os-release.html

> Lines beginning with "#" are treated as comments. Blank lines are permitted and ignored.

does above handle this requirement?

Reply:
Depending on needs it can be a broken symlink to unmounted /usr partition too.

> The file /etc/os-release takes precedence over /usr/lib/os-release. Applications should check for the former, and exclusively use its data if it exists, and only fall back to /usr/lib/os-release if that is missing.

Reply:
No - this seems like a likely culprit though. It might be worth making our own parser?

Reply:
That's interesting. The `osr.Parse` code doesn't seem to handle comments, that's probably our root cause.